renderer/components/Settings/SettingsForm.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { injectIntl, intlShape } from 'react-intl'
+import { Form } from 'components/UI'
+import messages from './messages'
+
+const SettingsForm = ({
+  intl,
+  configureAutoUpdater,
+  onSubmit,
+  setLocale,
+  saveConfigOverrides,
+  showNotification,
+  children,
+}) => {
+  const handleSubmit = async values => {
+    // Save the updated settings.
+    await saveConfigOverrides(values)
```

Comment:
Maybe it's worth to also handle submit exception and show a notif in that case?

Reply:
good call. Have added errors case